scripts/setup/setup-certbot

```diff
@@ -57,6 +53,14 @@ while true; do
     esac
 done
 
+# Parse the remaining arguments as Subject Alternative Names to pass to certbot
+shift
+HOSTNAMES=()
+for arg; do
+    HOSTNAMES+=("-d $arg")
```

Comment:
This should be `HOSTNAMES+=(-d "$arg")` (to add two strings to the array, not one whitespace-delimited string).